lib/protoparser/opentelemetry/firehose/http.go

```diff
@@ -12,11 +13,12 @@ import (
 func WriteSuccessResponse(w http.ResponseWriter, r *http.Request) {
 	requestID := r.Header.Get("X-Amz-Firehose-Request-Id")
 	if requestID == "" {
-		// This isn't a AWS firehose request - just return an empty response in this case.
+		// This isn't an AWS firehose request - just return an empty response in this case.
 		w.WriteHeader(http.StatusOK)
 		return
 	}
 
+	requestID = html.EscapeString(requestID)
 	body := fmt.Sprintf(`{"requestId":%q,"timestamp":%d}`, requestID, time.Now().UnixMilli())
```

Comment:
I'd name "timestamp" -> "timestamp_ms" if possible.

Reply:
we cannot rename this field, as it's required by AWS https://docs.aws.amazon.com/firehose/latest/dev/httpdeliveryrequestresponse.html#responseformat